draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+% Translator info: In Homam IV was Quote like: "Rufux will crush this army like a bug." If has scounting and has bigger army. So "like a bug" is little easter egg referencing on this.
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1--6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–-III) Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
+
+\textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}
+
+\subsection*{\MakeUppercase{AI Hero Setup}}
+
+Prepare an AI Deck for Settlement defenders.
+
+\textbf{Settlement defenders Deck:} 1 × Attack Card, 1 × Defense Card, 1 × Spell Card
+
+\textbf{Settlement defenders Spell Deck:} 1 × Magic Arrow
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Conquer all Settlements.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+\begin{itemize}
+  \item Any player loses their Town.
+  \item At least one Settlement remains unclaimed by the end of the \nth{7} Round. Time has run out!
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+    \textbf{\nth{1} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.
+    \end{itemize}
+    \textbf{\nth{2} Round:}
+    \begin{itemize}
+      \item  First wave of enemy clan armies attacks the towns. (see below)
+      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
+    \end{itemize}
+    \textbf{\nth{3} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
```

Comment:
```suggestion
\end{itemize}
\textbf{\nth{3} Round:}
\begin{itemize}
  \item At the start of each Round, each player reveals one Far Map Tile (II--III). This Tile may be revealed at no cost, whether it comes from the Pool or Map Template.
```

Reply:
same here

Reply:
Fixed. I wonder if those "At the start of the Round" are necessary. Those events happen at the start anyway?

Reply:
I think its clearier that on a start of round not start of each players turn... its minor think but for players is better to explore it at start to more posibilities.